tiledb/cloud/notebook.py

```diff
@@ -1,10 +1,23 @@
-from tiledb.cloud import array
-from tiledb.cloud import client
-from tiledb.cloud import rest_api
-from tiledb.cloud import tiledb_cloud_error
+"""
+Python support for notebook I/O on Tiledb Cloud. All notebook JSON content
+is assumed to be encoded as UTF-8.
+
+"""
+
+import posixpath
+import time
+from typing import Tuple
+
+import numpy
+
+import tiledb
+from tiledb.cloud import array, client, rest_api, tiledb_cloud_error
```

Comment:
`isort .`

Reply:
i have done so many times and i am now doing so on every commit.

* what formatting would you prefer?
* does isort give you something different?

Reply:
This was one of those comments where one of your follow-ons ended up fixing the problem and the comment became outdated; what appears in efdc478a30413c37d3ba466eab77c64621e497c4 is The Correct Format.